bundle/manifests/bpfman-operator.clusterserviceversion.yaml

```diff
@@ -215,7 +215,7 @@ metadata:
       ]
     capabilities: Basic Install
     containerImage: quay.io/bpfman/bpfman-operator:v0.0.0
-    createdAt: "2024-06-04T18:15:26Z"
+    createdAt: "2024-06-05T21:00:08Z"
```

Comment:
I would just drop this change, meaningless.

Reply:
removed

Reply:
dropped